test/e2e/suite_test.go

```diff
@@ -301,19 +301,21 @@ var _ = AfterSuite(func() {
 	By("Cleaning up Rancher Clusters")
 
 	for _, rancherCluster := range rancherClusterList.Items {
+		if rancherCluster.Name == "local" {
```

Comment:
Not yet familiar with the repo but just for my understanding: "local" cluster is the one we never run in e2e tests but hardcoded for local usage?

Reply:
it represents a management cluster and is created by Rancher